src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -341,21 +549,27 @@ values of the span limits
 * `OTEL_EVENT_ATTRIBUTE_COUNT_LIMIT`
 * `OTEL_LINK_ATTRIBUTE_COUNT_LIMIT`
 
-The following environment variables can be used to override the default
-values of the log record limits
-(following the [OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.25.0/specification/configuration/sdk-environment-variables.md#logrecord-limits)).
+The following environment variables can be used to configure log record limits
+used for logging:
 
 * `OTEL_LOGRECORD_ATTRIBUTE_VALUE_LENGTH_LIMIT`
 * `OTEL_LOGRECORD_ATTRIBUTE_COUNT_LIMIT`
 
-## Environment Variables for Experimental Features
+### Experimental features
```

Comment:
Make this a bigger heading.

```suggestion
## Experimental features
```

Reply:
Right now it is a sub-section of "Environment Variables" with the toc represented as:

  * [Environment Variables](#environment-variables)
    * [Exporter configuration](#exporter-configuration)
    * [Attribute limits](#attribute-limits)
    * [Experimental features](#experimental-features)

I can pop it out but it will become its own section and then probably needs some text that it is envvar-based. Is that what you want?

Reply:
We have two kinds of experimental features:
- Environment Variables/ Config based
- Public API changes that are enabled by default in pre-release builds

I think we should make "Experimental Features" its own section. The public API ones are not dependent on environment variables. Even though OTLP Exporter only has environment variable based experimental features other components such as SDK have both kinds of experimental features. It would be nice to stay consistent and have all docs contain a dedicated section for experimental features.

Reply:
Pushed updates: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5452/commits/727012a546d511abd0dc9f5e420e36bb52cdc180